pkg/apis/contrail/v1alpha1/cassandra_types.go

```diff
@@ -240,7 +240,21 @@ func (c *Cassandra) PodIPListAndIPMapFromInstance(instanceType string, request r
 
 // SetInstanceActive sets the Cassandra instance to active.
 func (c *Cassandra) SetInstanceActive(client client.Client, activeStatus *bool, sts *appsv1.StatefulSet, request reconcile.Request) error {
-	return SetInstanceActive(client, activeStatus, sts, request, c)
+	if err := client.Get(context.TODO(), types.NamespacedName{Name: sts.Name, Namespace: request.Namespace},
+		sts); err != nil {
+		return err
+	}
+	active := false
```

Comment:
Could activeStatus variable be used directly here instead of having additional local variable?

Reply:
fixed